### PR TITLE
Update Free Membership.js

### PR DESCRIPTION
**Pull request by andricngyh**
When I buy membership it is free

### CODE REVIEW

File: Player/Free Membership.js [hunk 1]
```diff
@@ -1 +1 @@
...
+Free membership
```
Comment (charlieh919) [inline Player/Free Membership.js:1]: [](url)

**Review by charlieh919 — APPROVED** [review]

**Review by michele200 — COMMENTED** [review]
how to get a free member ship

**Comment by rhys49** [comment]
vhgvgv

**Comment by rxzyx** [comment]
This is code, which is obfuscated, and not updatable

**Pull request closed without merging**